package.json

```diff
@@ -41,7 +41,7 @@
     "federalist": "npx gulp build",
     "lint": "gulp lintJS lintSass && npm run prettier:scss",
     "prestart": "npx gulp build",
-    "proof": "bundle exec htmlproofer --allow-hash-href=true --enforce-https=false --allow-missing-href=true --ignore-empty-alt=true --ignore-files '/_site/whats-new/updates/2017/'  --ignore-status-codes 403,429 ./_site",
+    "proof": "bundle exec htmlproofer --allow-hash-href=true --enforce-https=false --allow-missing-href=true --ignore-empty-alt=true --ignore-files '/_site/whats-new/updates/2017/' --ignore-status-codes 0,403,429,503 ./_site",
```

Comment:
Sites reported with status code 0 were working properly. These were `https://lincs.ed.gov/` and `https://www.flra.gov/`.

Codes 503 were redirects (fastcompany) and took too long (congress.gov/bill/115th-congress/house-bill/5759)